src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2442,6 +2442,15 @@ void ApplicationManagerImpl::UnregisterApplication(
     MessageHelper::SendStopAudioPathThru(*this);
   }
 
+#ifdef SDL_REMOTE_CONTROL
+  std::function<void(plugin_manager::RPCPlugin&)> on_app_unregistered =
```

Comment:
@BSolonenko actually, you could replace this long type declaration with `auto` as you did in all other places